lib/export.js

```diff
@@ -1792,12 +1792,24 @@ class FHIRExporter {
 
   checkMappedAndConstrained(profile, path) {
     const codeEl = common.getSnapshotElement(profile, path);
+    let fqn = profile.identifier[0].value.split(".");
+    let identifier = {"name": fqn.pop(),"namespace": fqn.join(".")};
+    const def = this._specs.dataElements.findByIdentifier(identifier);
+
     if (this.elementTypeNotChanged(profile, codeEl)) {
-      logger.warn('No mapping to \'%s\'. This property is core to the target resource and usually should be mapped.', path);
+      if (!def._isAbstract) {
+        logger.warn('No mapping to \'%s\'. This property is core to the target resource and usually should be mapped.', path);
+      } else {
+        logger.info('Abstract Class: No mapping to \'%s\'. This property is core to the target resource and usually should be mapped.', path);
```

Comment:
This is fine -- but I wonder if we need to report this situation at all.  I expect it may often be the norm for an abstract class not to have a mapping?